src/utils/helpers.js

```diff
@@ -14,28 +14,45 @@ const COMMON_KEYS = CONSTANTS.COMMON_KEYS;
 /** Traverses the children of a tree 
  * @param  {object} parent Object containing the children 
  * @param  {object} currentChildren Object containg the current directory tree
+ * @param  {string} includeOnlyRegExp String to match to include only certain paths
  * @return {object} currentChildren Updated object with new children added
  */
-let getChildren = (parent, currentChildren) => {
-  //If it has children, it is a directory
-  if (parent.children) {
-    //Push it into our directory array
-    currentChildren.children.push({
-      [COMMON_KEYS.PATH]: parent.path,
-      [COMMON_KEYS.LENGTH]: parent.children.length
-    });
+let getChildren = (parent, currentChildren, includeOnlyRegExp) => {
```

Comment:
It runs through this loop 4 times so it can't be a const because it'll be reassigned.